test/integration/vistor_can_view_items_test.rb

```diff
@@ -3,8 +3,8 @@
 class VistorCanViewItemsTest < ActionDispatch::IntegrationTest
 
   def create_items(num)
-    num.times do
-      Item.create(title: "latte", description: 'forthy deliciousness', price: 4.00, image: "google.com")
+    num.times do |i|
+      Item.create(title: "latte#{i}", description: 'forthy deliciousness', price: 4.00, image: "google.com")
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.